### PR TITLE
Finalisation project guides

### PR DESCRIPTION
**Pull request by JasonScourge**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -331,19 +331,6 @@ Exits the program.<br>
...
-#### Changing your save location : `saveto`
-To be edited.<br>
-Format: `exit`  
-
-[Back To Top](#user-guide)
-[//]: # (@@author)
-
-[//]: # (@@author A0139401N)
-### Saving the data 
-The Do*er*-list data are saved in the hard disk automatically after any command that changes the data.<br>
```
Comment (haiilong) [inline docs/UserGuide.md]: These are not unnecessary lines. Data are saved in the hard disk automatically after any command that changes the data. It is true.

Comment (haiilong) [inline docs/UserGuide.md]: The Diagrams_Model.pptx is necessary. They have the original images of our Developer Guide so future developer can easily change it.

Comment (JasonScourge) [inline docs/UserGuide.md]: Yeah, corrected the error. Meant to remove the repeated "Changing your save location".

Comment (JasonScourge) [inline docs/UserGuide.md]: The Diagrams_Model.pptx is not being removed; but there is a **ghost/temp** file of the Diagrams_Model.pptx that I have removed it, I think probably one of us has pushed it up accidentally.

**Comment by JasonScourge** [comment]
The UserGuide.md has no reference to part I, II, IV, etc. 
So it really looks weird if we placed the part I, II, II in the images. 
On the other hand, the other UserGuide for CS2101 side has it; so we are going to implement the images you have created for that part.

**Pull request merged**